packages/main/src/themes/Tokenizer.css

```diff
@@ -53,6 +53,7 @@
 }
 
 .ui5-tokenizer-more-text {
+	min-width: 3.25rem;
```

Comment:
According to the Fiori specification, the min-width of the MultiComboBox/MultiInput is 10.5rem. As we add min-width to the N-more label in order to not get cut off on small width, it is not valid because without that min-width it gets cut off on smaller size than the minimum component width. As a result, we should not add this, because it affects the positioning of inner input value.